debian/changelog

```diff
@@ -1,3 +1,16 @@
+cloud-init (24.1.3-0ubuntu3.2) noble; urgency=medium
+
+  * Upstream bug fix release based on 24.1.6
+    + functional fixes in debian/patches:
+      - cherry-pick 417ee551: fix(ec2): Ensure metadata exists before
+        configuring PBR. (LP: #2066979)
+      - cherry-pick d6776632: fix: Check renderer for netplan-specific code
+        (#5321). (LP: #2066985)
+    + test fixes in debian/patches:
+      - cherry-pick 74dc7cce: test: Fix failing test_ec2.py test (#5324)
```

Comment:
Since we are manually constructing this, let's compress it a bit to the actual filename prefix of the cpick- files that better directly reprsents the files SRU folks will have to review:
```suggestion
    + functional fixes in debian/patches:
      - cpick-417ee551: fix(ec2): Ensure metadata exists before configuring PBR.
         (LP: #2066979)
      - cpick-d6776632: fix: Check renderer for netplan-specific code (#5321)
        (LP: #2066985)
    + test fixes in debian/patches:
      - cpick-74dc7cce: test: Fix failing test_ec2.py test (#5324)
```